templates/pkg/resource/sdk.go.tpl

```diff
@@ -49,6 +49,13 @@ func (rm *resourceManager) sdkCreate(
 	ctx context.Context,
 	r *resource,
 ) (*resource, error) {
+{{ $customMethod := .CRD.GetCustomImplementation .CRD.Ops.Create }}
```

Comment:
Use `{{- ... -}}` here to remove the empty newline being generated

Reply:
Done